gosearch.go

```
@@ -63,64 +62,290 @@ var tlsConfig = &tls.Config{
+	go HudsonRock(username, &wg)
+	wg.Wait()
+
+	// Search Breach Directory if API key is provided
+	if *breachDirectoryAPIKey != "" || *breachDirectoryAPIKeyLong != "" {
+		if *breachDirectoryAPIKey != "" {
+			apikey = *breachDirectoryAPIKey
+		} else {
+			apikey = *breachDirectoryAPIKeyLong
+		}
+		fmt.Println(strings.Repeat("⎯", 85))
+		strings.Repeat("⎯", 85)
```

golangci-lint (1.64.8)

SA4017: Repeat doesn't have side effects and its return value is ignored

(staticcheck)